interfaces/cython/cantera/test/test_kinetics.py

```diff
@@ -480,6 +480,16 @@ def test_sticking_coeff_err(self):
                 ct.Interface("sticking_coeff_check.yaml", "Pt_surf", [gas])
 
 
+def check_raises(yaml, err_msg, line):
+    """Helper function for InputFileErrors"""
+    err_msg = [err_msg]
+    err_msg.append("InputFileError thrown by ")
+    err_msg.append(f"Error on line {line} of ")
+    for err in err_msg:
+        with pytest.raises(ct.CanteraError, match=err):
+            ct.Solution(yaml=yaml)
```

Comment:
Is there a way of doing this without calling `ct.Solution` 3 times for each case?